src/utils/components/Textarea.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+
+export default function Textarea({
+  children,
+  onChange,
+  className,
+  name,
+  value,
+  rows = 3,
+  label,
+  disabled,
+}: {
+  children: JSX.Element
```

Comment:
We have another option instead of typing the typescript for the textarea element.
We can use ComponentProps helper from react which takes a typeof component or a name of an element and return its type.
`type TextAreaProps = ComponentProps<"textarea">` will give us all valid textarea props.